### PR TITLE
conntracksync changes

### PR DESCRIPTION
**Pull request by leodotcloud**
https://github.com/rancher/rancher/issues/8207

### CODE REVIEW

File: conntracksync/conntrack/conntrack.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	replyDestinationPortIndex    = 10
+)
+
+// ListDNAT lists only DNAT conntrack entries
+func ListDNAT() ([]CTEntry, error) {
+	return cmdCTListDNAT()
+}
+
+// CTEntryCreate Addetes the given entry from the conntrack table
+func CTEntryCreate(e CTEntry) error {
```
Comment (ibuildthecloud) [inline conntracksync/conntrack/conntrack.go:50]: no used, delete

File: conntracksync/conntrack/conntrack.go [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+	if err := cmd.Run(); err != nil {
+		logrus.Errorf("error adding conntrack entry")
+		return err
+	}
+	return nil
+}
+
+// CTEntryDelete deletes the given entry from the conntrack table
+func CTEntryDelete(e CTEntry) error {
+
```
Comment (ibuildthecloud) [inline conntracksync/conntrack/conntrack.go]: delete whitespace

File: conntracksync/conntrack/conntrack.go (hunk 1 above)
Comment (leodotcloud) [reply conntracksync/conntrack/conntrack.go:50]: I am using it in tests: https://github.com/rancher/plugin-manager/pull/62/commits/8ac914b3bb3f9f0403c305430c4f53f1737bb5c5#diff-337277601dc0c2f3bef9d8e80441356cR35

**Comment by deniseschannon** [comment]
https://github.com/rancher/rancher/issues/8412

**Pull request merged**